win32_event_log/datadog_checks/win32_event_log/win32_event_log.py

```diff
@@ -8,6 +8,7 @@
 # stdlib
 import calendar
 from datetime import datetime, timedelta
+from uptime import uptime
```

Comment:
Can you pin `uptime` to 3.0.1 in the requirements file ? If you use the 
https://github.com/DataDog/integrations-core/tree/master/datadog_checks_dev cli utility, this can be easily done with `ddev dep pin uptime 3.0.1 win32_event_log` (cf https://github.com/DataDog/integrations-core/tree/master/datadog_checks_dev#pin)

Reply:
`uptime` has been pinned